test/evergreen.yml

```diff
@@ -5864,8 +5873,9 @@ buildvariants:
       WT_TOPDIR=$(git rev-parse --show-toplevel)
       WT_BUILDDIR=$WT_TOPDIR/cmake_build
       LD_LIBRARY_PATH=$WT_BUILDDIR
+      TESTUTIL_MSAN=1
     # We don't run C++ memory sanitized testing as it creates false positives.
```

Comment:
Could this comment be extended to cover why the "long_running" label is excluded?